.gitignore

```diff
@@ -61,6 +61,7 @@ CMakeLists.txt.user
 .idea
 /.tidytmp
 vcpkg*/
+*.log
```

Comment:
doing some graph build experiments and logging to .log. happens frequently, might as well put it in .gitignore